packages/cli/src/evaluation/test-runner/test-runner.service.ee.ts

```diff
@@ -197,6 +222,12 @@ export class TestRunnerService {
 		const testRun = await this.testRunRepository.createTestRun(test.id);
 		assert(testRun, 'Unable to create a test run');
 
+		// 0.1 Initialize AbortController
+		const abortController = new AbortController();
+		this.abortControllers.set(testRun.id, abortController);
+
+		const abortSignal = abortController.signal;
```

Comment:
This should probable be the first thing on this method. Otherwise if we receive a cancel request before this line is reached it won't be canceled